content/web/tutorials/simplest-example.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Simplest Example-Web WorldWind/NASA WorldWind"
+date: 2017-11-15T13:48:00-08:00
+draft: false
+header: Simplest Example
+listdescription: Shows the simplest way to get started with Web WorldWind.
+---
+
+## Simplest Example
+
+This step-by-step tutorial illustrates how to use Web WorldWind in an HTML file.
+
+---
+
+### Breakdown
+
+First, in the head tag of your HTML, add a script element to include the Web WorldWind library.
+
+    <script src="https://files.worldwind.arc.nasa.gov/artifactory/apps/web/worldwind.min.js" type="text/javascript"></script>
+
+Next, create an HTML5 canvas, defining its width and heighth.
+
+    <canvas id="canvasOne" width="1024" height="768">
```

Comment:
Best to make this this element self-closing: <canvas id="..."/>

Reply:
The canvas element also contains "Your browser does not support HTML5 Canvas" between the opening and closing tags.